### PR TITLE
Fix tests and phpstan

### PR DESCRIPTION
**Pull request by dereuromark**
- removed code smells in tests ( https://github.com/spryker-sdk/spryk/runs/2609301482?check_suite_focus=true )
- fixed phpstan level 8

### CODE REVIEW

**Review by stereomon — APPROVED** [review]
I don't see any issues with your changes.

File: tests/SprykerSdkTest/Spryk/Integration/Glue/Dependency/Client/AddGlueDependencyClientBridgeMethodTest.php [hunk 1]
```diff
@@ -136,8 +123,6 @@ public function testAddsGlueDependencyClientBridgeMethodOnlyOnce(): void
      */
     public function testAddsGlueDependencyClientBridgeMethodOnlyOnceOnProjectLayer(): void
     {
-        $this->expectException(SprykWrongDevelopmentLayerException::class);
-
         $this->tester->run($this, [
```
Comment (dereuromark) [inline tests/SprykerSdkTest/Spryk/Integration/Glue/Dependency/Client/AddGlueDependencyClientBridgeMethodTest.php:126]: This is the now failing line, being cloaked by the wrong position of expectException() which should only happen to 2nd run() call.

Comment (dereuromark) [reply tests/SprykerSdkTest/Spryk/Integration/Glue/Dependency/Client/AddGlueDependencyClientBridgeMethodTest.php:126]: @AsonUnique Can you please elaborate?

Comment (AsonUnique) [reply tests/SprykerSdkTest/Spryk/Integration/Glue/Dependency/Client/AddGlueDependencyClientBridgeMethodTest.php:126]: It seems that you are right, the `expectException()` call is wrongly positioned.

Comment (dereuromark) [reply tests/SprykerSdkTest/Spryk/Integration/Glue/Dependency/Client/AddGlueDependencyClientBridgeMethodTest.php:126]: Now it is not wrongly positioned anymore, as mentioned above.
It now shows an issue in code, though, that needs fixing.
Do you know why it fails (wrongly)?

Comment (AsonUnique) [reply tests/SprykerSdkTest/Spryk/Integration/Glue/Dependency/Client/AddGlueDependencyClientBridgeMethodTest.php:126]: I'm afraid it was too long ago and I hardly remember what is going on here. 
Maybe @limeeugenia could help you here?

Comment (gechetspr) [reply tests/SprykerSdkTest/Spryk/Integration/Glue/Dependency/Client/AddGlueDependencyClientBridgeMethodTest.php:126]: For me it looks like test name was wrong. IMO it tests that we can't add core related code on project level

Comment (dereuromark) [reply tests/SprykerSdkTest/Spryk/Integration/Glue/Dependency/Client/AddGlueDependencyClientBridgeMethodTest.php:126]: Check the other 3-5 places where the same code is executed and the tests seem to pass
I do agree that the message here seems confusing.

Comment (gechetspr) [reply tests/SprykerSdkTest/Spryk/Integration/Glue/Dependency/Client/AddGlueDependencyClientBridgeMethodTest.php:126]: Yep. But there is a `testAddsGlueDependencyClientBridgeMethodsOnProjectLayer` test that tests actually this case.
So IMO we can remove this failing test at all.

Comment (dereuromark) [reply tests/SprykerSdkTest/Spryk/Integration/Glue/Dependency/Client/AddGlueDependencyClientBridgeMethodTest.php:126]: Fair enough.

**Pull request merged**